animation_nodes/nodes/mesh/mesh_object_input.py

```diff
@@ -15,6 +15,7 @@ def create(self):
         self.newInput("Boolean", "Use Modifiers", "useModifiers", value = False)
         self.newInput("Boolean", "Load UVs", "loadUVs", value = False, hide = True)
         self.newInput("Boolean", "Load Vertex Colors", "loadVertexColors", value = False, hide = True)
+        self.newInput("Boolean", "Load Vertex Weights", "loadVertexWeights", value = False, hide = True)
         self.newInput("Boolean", "Load Custom Attributes", "loadCustomAttributes", value = False, hide = True)
```

Comment:
Not related to this patch, but we should consider making those into node options instead.

Reply:
That will make the node more cleaner, and also be good for more options in the future.